test/integration/pool_connect_test.lua

```diff
@@ -35,7 +35,10 @@ function g.test_identity()
 
     -- netbox.connect implicitly calls fiber.create(...)
     -- so there would be one context switch
-    t.assert_equals(csw1 + 1, csw2)
+    -- changed in 2.11: netbox.connect doesn't yield
+    if not helpers.tarantool_version_ge('2.11.0') then
```

Comment:
Shouldn't it be 2.11.2?

Reply:
See this discussion regarding the version: https://github.com/tarantool/vshard/pull/473#pullrequestreview-1936721959